homeassistant/components/roon/media_player.py

```diff
@@ -0,0 +1,452 @@
+    def volume_level(self):
+        """Return current volume level."""
+        return self._volume_level
+
+    @property
+    def is_volume_muted(self):
+        """Return mute state."""
+        return self._is_volume_muted
+
+    @property
+    def volume_step(self):
+        """.Return volume step size."""
+        return self._volume_step
+
+    @property
+    def supports_standby(self):
+        """Return power state of source controls."""
+        return self._supports_standby
+
+    @property
+    def state(self):
+        """Return current playstate of the device."""
+        return self._state
+
+    @property
+    def is_nowplaying(self):
+        """Return true if an item is currently active."""
+        return self.state == STATE_PLAYING
+
+    @property
+    def shuffle(self):
+        """Boolean if shuffle is enabled."""
+        return self._shuffle
+
+    @property
+    def repeat(self):
+        """Boolean if repeat is enabled."""
+        return self._repeat
+
+    def media_play(self):
+        """Send play command to device."""
+        self._server.roonapi.playback_control(self.output_id, "play")
+
+    def media_pause(self):
+        """Send pause command to device."""
+        self._server.roonapi.playback_control(self.output_id, "pause")
+
+    def media_play_pause(self):
+        """Toggle play command to device."""
+        self._server.roonapi.playback_control(self.output_id, "playpause")
+
+    def media_stop(self):
+        """Send stop command to device."""
+        self._server.roonapi.playback_control(self.output_id, "stop")
+
+    def media_next_track(self):
+        """Send next track command to device."""
+        self._server.roonapi.playback_control(self.output_id, "next")
+
+    def media_previous_track(self):
+        """Send previous track command to device."""
+        self._server.roonapi.playback_control(self.output_id, "previous")
+
+    def media_seek(self, position):
+        """Send seek command to device."""
+        self._server.roonapi.seek(self.output_id, position)
+        # Seek doesn't cause an async update - so force one
+        self._media_position = position
+        self.schedule_update_ha_state()
+
+    def set_volume_level(self, volume):
+        """Send new volume_level to device."""
+        volume = int(volume * 100)
+        self._server.roonapi.change_volume(self.output_id, volume)
+
+    def mute_volume(self, mute=True):
+        """Send mute/unmute to device."""
+        self._server.roonapi.mute(self.output_id, mute)
+
+    def volume_up(self):
+        """Send new volume_level to device."""
+        self._server.roonapi.change_volume(self.output_id, 3, "relative")
+
+    def volume_down(self):
+        """Send new volume_level to device."""
+        self._server.roonapi.change_volume(self.output_id, -3, "relative")
+
+    def turn_on(self):
+        """Turn on device (if supported)."""
+        if self.supports_standby and "source_controls" in self.player_data:
```

Comment:
We can invert this check and call media play and then return. Then we can outdent below.